spec.bs

```diff
@@ -208,13 +254,39 @@ report=].
 Issue: Unique internal value is not an exported definition. See
     <a href="https://github.com/whatwg/infra/issues/583">infra/583</a>.
 
+<h4 dfn-type=dfn>Debug scope</h4>
+A debug scope is a <a spec=HTML>unique internal value</a> that identifies which
+{{PAHistogramContribution}}s should be sent in the same [=aggregatable report=].
+
+<h4 dfn-type=dfn>Scoping details</h4>
+A scoping details is a [=struct=] with the following items:
+<dl dfn-for="scoping details">
+: <dfn>get batching scope steps</dfn>
+:: An algorithm returning a [=batching scope=]
+: <dfn>get debug scope steps</dfn>
+:: An algorithm returning a [=debug scope=]
+
+</dl>
+
+<h4 dfn-type=dfn>Debug details</h4>
+A debug details is a [=struct=] with the following items:
+<dl dfn-for="debug details">
+: <dfn>enabled</dfn> (default false)
+:: A [=boolean=]
+: <dfn>key</dfn> (default null)
+:: Null or an unsigned 64-bit integer. The key must be null if enabled is false.
+
+</dl>
+
 <h4 dfn-type=dfn>Contribution cache entry</h4>
 A contribution cache entry is a [=struct=] with the following items:
 <dl dfn-for="contribution cache entry">
 : <dfn>contribution</dfn>
 :: A {{PAHistogramContribution}}
 : <dfn>batching scope</dfn>
 :: A [=batching scope=]
+: <dfn>debug info</dfn>
+:: A [=debug scope=] or a [=debug details=]
```

Comment:
Wdyt if we have separate fields for debug scope and debug details to be more clear?